andino_gz/models/office/model.sdf

```diff
@@ -83,7 +165,28 @@
         </material>
       </visual>
     </link>
-    
+    <link name='roof'>
+      <visual name='visual'>
+        <geometry>
+          <mesh>
+            <scale>1 1 1</scale>
+            <uri>meshes/roof.dae</uri>
+          </mesh>
+        </geometry>
+        <material>
+          <diffuse>1 1 1 1</diffuse>
+          <specular>0 0 0 0</specular> 
+          <pbr>
+            <metal>
+              <albedo_map>materials/textures/roof/TCom_Scifi_Panel_512_albedo.tif</albedo_map>
+              <normal_map>materials/textures/roof/TCom_Scifi_Panel_512_normal.tif</normal_map>
+              <roughness_map>materials/textures/roof/TCom_Scifi_Panel_512_roughness.tif</roughness_map>
```

Comment:
We launching the world I see a lot of: 
```
[ign gazebo-1] TIFFFieldWithTag: Internal error, unknown tag 0xa302.
```
errors.

Probably there is some not entirely supported tag within the `tif`s file.

Don't you have a jpg or png variant for these? We can move forward either way but just to be sure

Reply:
I would just try to convert that tif file to png/jpg and see how it looks (and if there is no more warnings).
If the looks is the same then we can keep those jpg.

Reply:
@larodriguez22 Just one comment to go!

Reply:
I pushed [d8f10ec](https://github.com/Ekumen-OS/andino_gz/pull/84/commits/d8f10ec074b88863e67fcb657f30eaf33eab1d57) addressing that. Try it locally and let me know how it looks!